CHANGELOG.md

```diff
@@ -7,10 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.0] - 2019-02-24
+
+This release adds support for (protocol buffers in rippled)[https://github.com/ripple/rippled/tree/develop/src/ripple/proto/rpc/v1]. The `Signer`'s `signTransactionMethod` previously returned only the signed bytes. After this change, it returns the entire signed and encoded transaction. This encapsulates the serialization and signing logic inside of this library, rather than forcing clients to understand it. This is a breaking change (See 'Breaking Changes') below. 
```

Comment:
It says `signTransactionMethod` here, but `signTransaction` below.

And up here it says that it previously returned only the signed bytes, but below it says `signTransaction` **now** returns an array of signed bytes.